### PR TITLE
Bug - Azure Sentinel -  Update fetch-incidents to support first fetch after upgrading the integration

### PR DESCRIPTION
**Pull request by MLainer1**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/47457

## Description
Update fetch-incidents to support first fetch after upgrading the integration, as I originally did not take into account that the last_run dict will not be empty upon upgrading.

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/AzureSentinel/ReleaseNotes/1_3_4.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### Azure Sentinel
+- Fixed an issue where the command ***fetch-incidents*** failed to fetch relevant incidents upon updating the integration.
```
Comment (dantavori) [inline Packs/AzureSentinel/ReleaseNotes/1_3_4.md:4]:
```suggestion
- Fixed an issue where the ***fetch-incidents*** command fetched old incidents after upgrading to the previous pack release.
```

**Review by dantavori — COMMENTED** [review]
looks good, please add a unittest

File: Packs/AzureSentinel/Integrations/AzureSentinel/AzureSentinel_test.py [hunk 2]
```diff
@@ -1255,6 +1255,31 @@ def test_process_incidents(self, args, client, expected_result):
...
 
+    def test_last_run_in_fetch_incidents(self, mocker):
+        """
+        Given: - AzureSentinel client, last_fetched_ids array, last_run, first_fetch_time, and a minimum severity.
+
+        When:
+            - Calling the fetch_incidents command.
+
+        Then:
+            - Validate the call_args had the correct filter.
```
Comment (ShahafBenYakir) [inline Packs/AzureSentinel/Integrations/AzureSentinel/AzureSentinel_test.py]: Let's be a bit more specific here: what exactly is the last_fetched_ids (empty list), and what exactly are we expecting to see in the filter

**Review by ShahafBenYakir — APPROVED** [review]
Nice work. See my docstring comment.

**Pull request merged**